### PR TITLE
HZN-1479: Enable instant refresh in topology generator based on the recent changes in the topology code

### PR DESCRIPTION
**Pull request by patrick-schweizer**
* JIRA: https://issues.opennms.org/browse/HZN-1479

Our [continuous integration system](http://bamboo.internal.opennms.com:8085) will test and verify your changes.

Does not work yet - there are dependency problems in Karaf

### CODE REVIEW

File: container/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -1363,6 +1363,7 @@
     <feature name="opennms-enlinkd-shell" description="OpenNMS :: Features :: Enlinkd :: Shell" version="${project.version}">
         <bundle>mvn:org.opennms.features.enlinkd/org.opennms.features.enlinkd.shell/${project.version}</bundle>
         <bundle>mvn:org.opennms.features.enlinkd/org.opennms.features.enlinkd.generator/${project.version}</bundle>
+        <bundle>mvn:org.opennms.features.topology.plugins.topo/org.opennms.features.topology.plugins.topo.linkd/${project.version}</bundle>
```
Comment (patrick-schweizer) [inline container/features/src/main/resources/features.xml]: not sure if we need this dependency - it doesn't solve the Karaf dependency issue (see https://issues.opennms.org/browse/HZN-1479?focusedCommentId=53785&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-53785)

Comment (patrick-schweizer) [inline container/features/src/main/resources/features.xml]: not sure if we need this dependency - it doesn't solve the Karaf dependency issue (see https://issues.opennms.org/browse/HZN-1479?focusedCommentId=53785&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-53785)

**Comment by mattixtech** [comment]
@patrick-schweizer Hey Patrick, Just FYI so we don't end up duplicating effort. I am currently working on updating the generator code you wrote to call into the OnmsTopologyDao that Antonio wrote to notify it of all the changes the generator made and so these can be pushed out via Kafka.
https://github.com/OpenNMS/opennms/tree/mbrooks/OCE-40

Doesn't look like that is covered by your work in this PR but just wanted to let you know in case you were going to look into that.

**Comment by patrick-schweizer** [comment]
@mattixtech my goal is here to see the changes from the TopologyGenerator immediatledly in the topology map. I assume you want to do the same? If so I will close this PR :-)

**Pull request closed without merging**